src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,583 @@
+* Chrome 31+ 
+* Chrome for Android 40+
+* Safari 7.1+
+* iOS Safari 7.1+
+
+## Creator ##
+
+Use the following function to create a Text To Speech component:
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>fluid.textToSpeech(options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                Instantiate the text to speech component. The methods of the instantiated component can be called to queue up and read texts.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>options</dt>
+                    <dd>
+                        An optional data structure that configures the Text To Speech component, as described below.
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <th>Returns</th>
+            <td>The Text To Speech component</td>
+        </tr>
+        <tr>
+            <th>Examples</th>
+            <td>
+<pre>
+<code>
+var tts = fluid.textToSpeech({
+    utteranceOpts: {
+        volume: 1
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <th>See also</th>
+            <td>
+                <a href="#utteranceopts-options">Utteranceopts Options</a>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Supported Methods ##
+
+### `queueSpeech` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.queueSpeech(text, interrupt, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>queueSpeech</code> method allows you to append the text to the end of the queue to begin speaking after the other texts in the queue have been spoken. Setting the `interrupt` argument to true will remove all texts from the queue before adding the text to the queue.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>text</dt>
+                    <dd>
+                        The string of the text to be appended to the end of the queue to be spoken.
+                    </dd>
+                    <dt>interrupt</dt>
+                    <dd>
+                        An optional boolean value. The default value is false. Setting it to true will remove all texts from the queue before adding the text. Setting it to false does not interfere the component state.
+                    </dd>
+                    <dt>options</dt>
+                    <dd>
+                        An optional javascript object. To config the internal `SpeechSynthesisUtterance` instance. Anything that can go in <a href="#utteranceopts-options">utteranceOpts</a> can also be used as an option to this method.
```

Comment:
"SpeechSynthesisUtterance" should be in `<code>` block. 

Also we may want to rephrase this like:

"An optional javascript object. Allows for the configuration of the specific <code>SpeechSynthesisUtterance</code> instance used for this particular text. The configuration passed in here takes the same form as <a href="#utteranceopts-options">utteranceOpts</a> and will override them for this instance only."